style.css

```diff
@@ -1 +1,239 @@
+
+}
+.logo {
+    width: 25%;
+    height: auto;
+
+}
+.logo-img {
+    height: 120px;
+    width: 120px;
+}
+.header-side-contents {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-end;
+    padding-right: 2rem;
+    text-align: right;
+    width: 50%;
+}
+/* .................header style finish .............. */
+
+
+
+/* ........navigation style start........  */
+nav {
+    grid-column: 1/-1;
+    display: none;
+}
+
+nav>ul {
+    
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+
+    
+}
+
+li {
+    list-style: none;
+    font-style: normal;
+    margin: 2%;
+    background: #E8B5B8;
+    padding: 1rem;
+}
+a {
+    text-decoration: none;
+    font-size: 1.25rem;
+    font-family: 'Courier New', Courier, monospace;
+
+}
+/* .........navigation finish......................  */
+
+
+
+/* .......welcome/intro style start...... ............ */
+.intro-texts {
+    grid-column: 1/-1;
+    padding: 0 3%;
+    letter-spacing: 1px;
+    font-size: 1.5rem;
+
+    
+
+}
+/* .......welcome/intro style finish...... ............ */
+
+
+/* ..........main/big image div style start..........  */
+.big-image {
+    height: 300px; 
+    width: 100vw;
+    background-image: url("../images/cake2.jpg");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    grid-column: 1/-1;
+    grid-row: 3/6;
+
+    
+}
+/* ..............big image style finish..........  */
+
+
+/* ...........small images div style start.......  */
+.small-images {
+    grid-column: 1/-1;
+    background: pink;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+    align-items: space-between;
+    margin-bottom: 2rem;
+}
+
+.small-img {
+    width: 200px;
+    height: 200px;
+    margin: 1rem;
+}
+/* ........small images style finish.......  */
+
+
+/* ..........footer style start....... .........  */
+footer {
+    grid-column: 1/-1;
+    background: #E8B5B8;
+    font-size: 1.25rem;
+    padding: 5%;
+    display: flex;
+    flex-direction: column;
+}
+.social {
+    display: flex;
+    flex-direction: column;
+    
+}
+.copyright-text {
+    font-family: 'Courier New', Courier, monospace;
+    font-size: 15px;
+    text-align: right;
+    opacity: 0.7;
+
+}
+/* ..........footer style finish..........  */
+
+
+
+/* tablet style  */
+@media (min-width: 540px) {
+    nav {
+        display: block;
+    }
+    .hamburger {
+        display: none;
+    }
+    
+    
+}
+
+/* desktop style  */
+@media (min-width: 1040px) {
+    .social {
+        flex-direction: row;
+        justify-content: space-between;
+        width: 50%;
+        margin-left: 20%;
+        margin-right: 20%;
+        align-items: center;
+    }
+    
+    header {
+        
+        height: 200px;
+    }
+    
+    nav {
+       grid-row: 2/3; 
+    }
+
+    .big-image {
+        grid-column: 1/2;
+        height: 300px;
+        width: 60vw;
```

Comment:
This is the same as in the desktop version, .big-image is 60vw and .intro-texts is also 40vw so overall you have your 100vw plud padding, hence the scroll bar.